spk/comskip/src/installer.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
+
+# Package
+PACKAGE="comskip"
+DNAME="Comskip"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+PATH="${INSTALL_DIR}/bin:${PATH}"
+
+
+preinst ()
+{
+    exit 0
+}
+
+postinst ()
+{
+    # Link
+    ln -s ${SYNOPKG_PKGDEST} ${INSTALL_DIR}
+
+    # Put comskip in the PATH
+    mkdir -p /usr/local/bin
+    ln -s ${INSTALL_DIR}/bin/comskip /usr/local/bin/comskip
+
+    exit 0
+}
+
+preuninst ()
+{
+    exit 0
+}
+
+postuninst ()
+{
+    # Remove link
+    rm -f ${INSTALL_DIR}
+    rm -f /usr/local/bin/comskip
+
+    exit 0
+}
+
+preupgrade ()
+{
+    exit 0
```

Comment:
since you copy the config file to `var` folder... when one changes those, you should backup/restore the var folder here... see: https://github.com/SynoCommunity/spksrc/blob/master/spk/syncthing/src/installer.sh#L107-L109